### PR TITLE
Lovisas todo

### PR DESCRIPTION
**Pull request by LovisaBrorson**
Link to my deployed project: https://lovisas-todo.netlify.app/

### CODE REVIEW

File: code/src/components/TodoList.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  border-radius: 10px;
+  padding: 10px;
+  font-size: 10px;
+  margin-top: 7px;
+  cursor: pointer;
+`
+
+const TodoList = () => {
+  const items = useSelector((store) => store.todos.items)
+  console.log(items)
```
Comment (Mimmisen) [inline code/src/components/TodoList.js]: Here you forgot to remove a console log :)

File: code/src/index.css [hunk 2]
```diff
@@ -1,13 +1,125 @@
...
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+.todo-container {
+  height: 430px;
+  background-color: rgb(247, 230, 197);
+  border-radius: 25px;
+  padding: 16px 35px 16px 35px;
```
Comment (Mimmisen) [inline code/src/index.css]: on rows 12 and 33 you write the padding/margin px four times, if the value is the same on top and bottom, and right and left you only need to write `padding: 16px 35px` that covers all sides :)

File: code/src/index.css [hunk 3]
```diff
@@ -1,13 +1,125 @@
...
+
+  [type="checkbox"]:not(:checked) {
+    position: absolute;
+    left: 35px;
+  }
+
+  [type="checkbox"]:checked {
+    position: absolute;
+    left: 35px;
+  }
```
Comment (Mimmisen) [inline code/src/index.css:133]: You don't need the `positon:absolute `here in the media queries, you already have them in the original part of the code :)

**Review by Mimmisen — APPROVED** [review]
Awesome work with your project Lovisa! 🥇 👍 
You todo works great and you fulfill all blue level requirements and have also done some of the red ones!  Your code is easy to read and follow and you have done overall a great job! 
Ha det gött, hej :)

**Comment by LovisaBrorson** [comment]
Tack för feedback! Jag har snyggat till koden efter dina förslag! :)
Ha det gött, heeej!

Den tis 19 apr. 2022 kl 17:48 skrev Mimmi Fordal Uddin <
***@***.***>:

**Pull request closed without merging**